lib/App/Sqitch/Engine.pm

```diff
@@ -249,6 +249,11 @@ sub deploy {
         )
     );
 
+    $sqitch->info(__ "Will deploy the following changes:");
```

Comment:
I think this should be tied to a flag. Or maybe just emit this information if `$sqitch->verbosity > 1`) (1 is the default).

Reply:
How is this?

Reply:
Ah, the `debug` method, very good!